src/main.ts

```diff
@@ -1,13 +1,37 @@
-import { enableProdMode } from '@angular/core';
-import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
+import { withInterceptorsFromDi, provideHttpClient } from '@angular/common/http';
+import { enableProdMode, importProvidersFrom } from '@angular/core';
+import { BrowserModule, bootstrapApplication } from '@angular/platform-browser';
+import { provideRouter } from '@angular/router';
+import { EffectsModule } from '@ngrx/effects';
+import { StoreModule } from '@ngrx/store';
+import { HttpClientInMemoryWebApiModule } from 'angular-in-memory-web-api';
 
-import { AppModule } from './app/app.module';
+import { AppComponent } from './app/app.component';
+import { appRoutes } from './app/app.routes';
+import { extModules } from './app/build-specifics';
+import { ProductData } from './app/products/product-data';
+import { ProductEffects } from './app/products/state/product.effects';
+import { productReducer } from './app/products/state/product.reducer';
+import { userReducer } from './app/user/state/user.reducer';
 import { environment } from './environments/environment';
 
 if (environment.production) {
     enableProdMode();
 }
 
-platformBrowserDynamic()
-    .bootstrapModule(AppModule)
-    .catch((err) => console.error(err));
+bootstrapApplication(AppComponent, {
+    providers: [
+        provideRouter(appRoutes),
+        provideHttpClient(withInterceptorsFromDi()),
+        importProvidersFrom(
+            BrowserModule,
+            StoreModule.forRoot({}),
+            StoreModule.forFeature('users', userReducer),
+            StoreModule.forFeature('products', productReducer),
```

Comment:
The original app module does not register feature states yet

<img width="509" alt="image" src="https://github.com/user-attachments/assets/1a3521b1-78a7-4b71-a9e6-abc6d8ac8732" />


The feature states and effects should be initialised with the corresponding routes:
<img width="778" alt="image" src="https://github.com/user-attachments/assets/1dd8713f-e369-4b21-8de1-123fa26ba374" />

See NgRx article: https://dev.to/ngrx/using-ngrx-packages-with-standalone-angular-features-53d8

Reply:
FYI this syntax was not conclusive yet and has been changed to:
- `provideState` instead of `provideStoreFeature`
- `provideEffects` instead of `provideFeatureEffects`